### DISCUSSION TITLE
Access to close Issues and PR's

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#135] [General]
Hey @ThioJoe ,

Would it be possible for you to give me access to close issue threads and PR's of other people.

Thanks,
Kendall Does Coding

**Comment by TechStudent10** [comment]
This is the equivalent of asking for mod in a Discord server. You can't just ask and expect a "yes".

This is also a big privilege. You are asking for write access which allows you to add commit, merge PRs, etc. So I really don't think it's even gonna be a yes.

**Comment by KendallDoesCoding** [comment]
Yeah, I understand that ,That's totally fine!
